modules/ximgproc/include/opencv2/ximgproc.hpp

```diff
@@ -56,26 +56,21 @@
 #include "ximgproc/fourier_descriptors.hpp"
 #include "ximgproc/ridgefilter.hpp"
 
-
 /** @defgroup ximgproc Extended Image Processing
   @{
     @defgroup ximgproc_edge Structured forests for fast edge detection
 
 This module contains implementations of modern structured edge detection algorithms,
 i.e. algorithms which somehow takes into account pixel affinities in natural images.
 
-    @defgroup ximgproc_edgeboxes EdgeBoxes
```

Comment:
Changes in this file looks unrelated to PR purpose. Are these changes necessary?

Reply:
@tenderfail I added the interface to the brightedge method if any one wants to use it just as the other  do to expose their methods from the src folder. I can leave this hpp alone if you wish.

Reply:
Lines changed in the file are related to the documentation structure. Also removed documentation groups are used in descriptions of other algorithms so the change could even affect documentation build. Thus IMHO these changes should be reverted.